src/components/ItaliaTheme/Blocks/ContactsBlock/Edit.jsx

```diff
@@ -106,15 +106,9 @@ class Edit extends SubblocksEdit {
             </div>
 
             <SubblocksWrapper node={this.node}>
-              <Row
-                className={
-                  this.state.subblocks.length > 3
-                    ? 'justify-content-start'
-                    : 'justify-content-center'
-                }
-              >
+              <Row className="justify-content-center">
                 {this.state.subblocks.map((subblock, subindex) => (
-                  <Col lg="4" key={subblock.id} className="pb-3">
```

Comment:
La classe pb-3 va lasciata perchè nel caso in cui ci siano piu di 3 blocchi e quindi vanno a capo devono rimanere distanziati e non appiccicati